invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/frontpage.less

```
@@ -20,9 +20,12 @@ h2 {
   margin-inline-end: 0px;
 }
 
+
+.ui.label.blue,
```

also target the blue.label
```suggestion
.ui.blue.labels .label, .ui.blue.label 
```